src/components/layout/HeroHeader/index.tsx

```diff
@@ -14,14 +14,18 @@ type InstallOptionProps = Card & {
 };
 
 const detectOperatingSystem = () => {
-  return window.navigator.userAgent
-    .toLowerCase()
-    .split(' ')
-    .find(item => item.includes('linux' || 'windows' || 'macos'));
+  const userAgent = window.navigator.userAgent.toLowerCase().split(' ');
+  if (userAgent.find(item => item.includes('windows'))) {
+    return 'windows';
+  } else if (userAgent.find(item => item.includes('macos'))) {
```

Comment:
Per the User-Agent values in https://github.com/containers/website-new/issues/43#issuecomment-1534918133 , `macos` is not going to match.